temba/channels/types/firebase/type.py

```diff
@@ -31,6 +31,13 @@ class FirebaseCloudMessagingType(ChannelType):
     max_length = 10000
     attachment_support = False
     free_sending = True
+    quick_reply_text_size = 36
+
+    @classmethod
+    def format_quick_replies(cls, quick_replies):
```

Comment:
as above for the facebook channel - it seems odd that the arg here is called quick_replies when really it's a serialised version of Msg.metadata